### PR TITLE
feat: `ModalPickOne` component

### PR DESCRIPTION
**Pull request by ghost**
Screenshot_2020-06-17 Storybook(2)

### CODE REVIEW

**Review by lilienblum — APPROVED** [review]

File: src/components/Modal/component.stories.tsx [hunk 1]
```diff
@@ -17,10 +19,34 @@ export const Default = () => {
...
+  return (
+    <Modal open={true} onClose={() => {}}>
+      <Modal.Scroll>{items}</Modal.Scroll>
+    </Modal>
+  );
+};
+
+export const Bare = () => {
+  return (
+    <Modal open={true} onClose={() => {}}>
```
Comment (ghost) [inline src/components/Modal/component.stories.tsx:48]: Why function is required? Just don't call it if it's undefined?

**Review by ghost — APPROVED** [review]
The rest looks fine

File: src/components/Modal/component.stories.tsx (hunk 1 above)
Comment (ghost) [reply src/components/Modal/component.stories.tsx:48]: It's not required, actually. I just left in the example for some reason.

**Pull request merged**